packages/account/src/Configs/poa-common-mistake-examples-config.ts

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import IcDocumentNameMismatch from '../Assets/ic-document-name-mismatch.svg';
+import IcDocumentAddressMismatch from '../Assets/ic-document-address-mismatch.svg';
+import IcOldIssuedDocument from '../Assets/ic-old-issued-document.svg';
+import IcBlurryDocument from '../Assets/ic-blurry-document.svg';
+import IcCroppedDocument from '../Assets/ic-cropped-document.svg';
+import IcEnvelope from '../Assets/ic-envelop.svg';
+
+type TExampleImageConfig = {
+    image: React.ComponentType<React.SVGAttributes<SVGElement>>;
+    description: string;
+    ref: string;
```

Comment:
I think it is not required. Because the ref is custom value provided in the config. It has not business significance. It can be given string as it is not exported to use in other places

Reply:
Alrighty